test/models/ui/validation/new_comment_validation_test.dart

```diff
@@ -0,0 +1,69 @@
+import "package:flutter_test/flutter_test.dart";
+import "package:proxima/models/ui/validation/new_comment_validation.dart";
+
+void main() {
+  group("New Comment Model Testing", () {
+    test("hash overrides correctly", () {
+      // Define the expected result
+      final newCommentState = NewCommentValidation(
+        contentError: "contentError",
+        posted: true,
+      );
+
+      final expectedHash = Object.hash(
+        newCommentState.contentError,
+        newCommentState.posted,
+      );
+
+      final actualHash = newCommentState.hashCode;
+
+      expect(actualHash, expectedHash);
+    });
+
+    test("equality overrides correctly", () {
+      // Define the expected result
+      final newCommentState = NewCommentValidation(
+        contentError: "commentError",
+        posted: true,
+      );
+
+      final newCommentStateCopy = NewCommentValidation(
+        contentError: "commentError",
+        posted: true,
+      );
+
+      expect(newCommentState, newCommentStateCopy);
+      expect(newCommentState == newCommentStateCopy, true);
```

Comment:
I think that is line is also a duplicate of the previous line.

Reply:
Fixed with [5141c90](https://github.com/ProximaEPFL/proxima/pull/358/commits/5141c90560386380b3d26de95338fdc3622b7d00)